setup.py

```diff
@@ -7,9 +7,9 @@
     install_requires=[
         "sqlalchemy",
         "psycopg2-binary",
-        "pandas<1.1",
+        "pandas",
         "numpy",
-        "streamlit==1.8.1",
```

Comment:
If we don't specify the version of the package here, it automatically installs the most recent version?

Reply:
yea that's correct